crates/cxx-qt-lib/src/types/qvariant/mod.rs

```diff
@@ -96,7 +96,7 @@ impl QVariant {
     /// if QVariant::canConvert is true otherwise returns None
     pub fn try_value<T: QVariantValue>(&self) -> Option<T> {
```

Comment:
This function should now be named `value`. Should be the default to use for the Rust API.

Reply:
Done